internal/presenter/packages/spdx_helpers.go

```diff
@@ -29,6 +30,52 @@ func getSPDXExternalRefs(p *pkg.Package) (externalRefs []spdx22.ExternalRef) {
 	return externalRefs
 }
 
+func getSPDXFiles(packageSpdxID string, p *pkg.Package) (files []spdx22.File, fileIDs []string, relationships []spdx22.Relationship) {
+	files = make([]spdx22.File, 0)
+	fileIDs = make([]string, 0)
+	relationships = make([]spdx22.Relationship, 0)
+
+	pkgFileOwner, ok := p.Metadata.(pkg.FileOwner)
+	if !ok {
+		return files, fileIDs, relationships
+	}
+
+	for _, ownedFilePath := range pkgFileOwner.OwnedFiles() {
```

Comment:
I just realized that the OwnedFiles is a full shotgun blast and has support from to exclude certain things.
https://github.com/anchore/syft/blob/9fe1da8ee60e128a5a61f406c0a03e178f4287d4/syft/pkg/ownership_by_files_relationship.go#L63-L67

I'm open to updating the signature so we can make the function smarter and have it only return a filtered list, but that means `matchesAny` gets repeated across implementations. Maybe there is a good middle ground here.

Reply:
@wagoodman Any opinions on where we want to go here? You have a deeper knowledge of Cataloger so I'll defer to you.

Do you think we should just repeat this logic in presenter or push the onus of `matchesAny` onto the respective `OwnedFiles` implementations?

Reply:
we went into a deeper answer on this offline; the short answer is this block doesn't apply to the relationship being added.